### ISSUE TITLE
regard/describe/portray... + as

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#387]
Shouldn't the as-phrase be an xcomp, as it is a predicative complement?

- I portray him as a villain: xcomp(describe, villain), with enhanced relation E:nsubj:xsubj(villian, him)
- The outcome is regarded as uncertain: xcomp(regarded, uncertain), E:nsubj:xsubj(uncertain, outcome)
- We regard the outcome as being uncertain: xcomp(outcome, uncertain), (in principle) E:nsubj:pass:xsubj(uncertain, outcome)

With this [rough query for as+ADJ](http://universal.grew.fr/?custom=63deb216c7c34), we see mostly `obl` in EWT and GUM.

For a handful of predicates in this class, [this query for as+NOUN](http://universal.grew.fr/?custom=63deb35642b16) shows mostly `obl` as well.

**Comment by amir-zeldes** [comment]
No, I think it's true that we have a secondary predication here from a semantic perspective (indeed, the coref layer in GUM labels the relationship between 'him' and 'villain' as link type `pred`), but syntactically I thought we were treating 'as' as a perposition (IN, ADP), and therefore it's just `obl`. Otherwise we would need to reconsider cases with other prepositions:

  * Do you take me **for** a fool?
  * You have a friend **in** me
  * It turned **to** dust

I think we have been treating all of these as `obl`, and only using `xcomp` if there is no preposition (become ->xcomp dust).

**Comment by nschneid** [comment]
"As" is a syntactically special preposition. For example, in this use it marks adjectives or even other PPs ("regard him as uncertain", "describe him as over his head").

"Like" overlaps in meaning:

- treat him as/like an assistant

But it doesn't take the same syntactic range of complements:

- treat him as/*like special
- designate him as/*like under consideration

**Comment by amir-zeldes** [comment]
I agree 'as' is special in many ways, but UD annotation has not so far distinguished it in trees. I also think the 'secondary predication' thing is only part of what makes it special, and is common to the uses of 'for', 'in' etc. above, so I don't think we should be singling out 'as' here. Notice also that 'as' can appear in an `nmod` function, which would result in adnominal `xcomp` in cases like "Dinner as a time of day". Under a simple `nmod` analysis this is unproblematic.

**Comment by nschneid** [comment]
OK, I should clarify that I am talking here about predicative complements as a grammatical function in clauses. "As" is a common way to mark them. (You've raised some interesting idioms like "take me for", but "for" doesn't regularly mark predicative complements, so that would be a borderline case. I'm also not talking about "as" in nominals, though the meaning may be similar.)

We see it is optional in cases like

- I consider you (as) my best friend.
- The judge declared the evidence (as) inadmissible.

Adjectives are particularly indicative because they are not generally marked by prepositions (with some special exceptions).

With other verbs, such as regard/describe/portray, "as" is mandatory on a nominal or adjectival predicative complement.

The [guidelines for secondary predicates](https://universaldependencies.org/u/overview/complex-syntax.html#secondary-predicates) do not give any examples with prepositions, but they do say "In UD, obligatory predicatives are always treated as an `xcomp`". [I take "predicatives" to be in the syntactic sense, i.e. constituents in a syntactic construction which regularly signals predication.] And this is reflected in prepositional `xcomp`s in both [GUM](http://universal.grew.fr/?custom=63e02954a9041) and [EWT](http://universal.grew.fr/?custom=63e0296718e3e), e.g. "leaving the prosecution's case [in turmoil]". (I would likewise expect "become" to have `xcomp` even with PPs—e.g. "the governor became out of touch"—though this is rare.)

So, at minimum I see a tension between the data and the guidelines. Maybe @dan-zeman or @jnivre has an opinion.

**Comment by amir-zeldes** [comment]
I don't see a fundamental difference between the "as" and "for" cases - "as" is just much more specialized for this function, but both prepositions can express a secondary predication in cases like "take X for Y", and it's just as obligatory in the appropriate reading: Do you take me (for a fool/??0)?

I could imagine a version of UD where all of these are xcomp, but that is not the case in almost any UD treebank I looked at in any language:

  * German HDT: 
    * als (=as): "betrachtet X als Y" `obl`
    * für (=for): "hält X für Y" `obl`
  * Polish PDB:
    * jako (=as): `obl`
    * mieć X za Y (=take X for Y, lit. 'behind') `obl`
  * Hebrew HTB:
    * ke (כ) (=as): `obl`
    * le (ל) (=to, e.g. חשב את X ל Y - lit. consider X to a Y): `obl`
  * Japanese GSDLUW:
    * として (=as): `obl` 

The one possible exception I found was Arabic PADT, which varies between the behavior above and, in the VERB attached case, sometimes having `xcomp` + `case` (for ك), where the latter should then arguably be `mark`. The adnominal case is consistently `nmod`.

In sum, even though I like being able to find secondary predications of all kinds, I think changing this in English would make it an outlier and distance it from other UD TBs, and is not trivial to do for all relevant prepositions (incl. for, to and maybe others) without manual effort.

**Comment by nschneid** [comment]
So, I guess there are two theoretical questions for the broader group: 1) what exactly counts as an "obligatory predicative", and 2) if PPs can be obligatory predicatives, should the xcomp criteria make an exception to exclude them because they bear non-core-like grammatical encoding?

**Comment by dan-zeman** [comment]
> Polish PDB:
> * jako (=as): `obl`

In Czech, _jako_ "as" phrases are among the most frequent types of secondary predication ("verbal attribute" in the PDT annotation guidelines). I suppose it is the same thing as the Polish _jako_, hence if the Polish data were annotated by the same people, it would end up annotated the same way.

The problem is that my conversion cannot handle this properly and it translates the original PDT deprel into `xcomp` even in the case of optional depictives, which are attached to one of the arguments of the verb rather than the verb itself. So searching for _jako_-secondary predicates in Czech (http://hdl.handle.net/11346/PMLTQ-UTQG) will return many wrong trees. It should be never `xcomp` if it is adnominal.

**Comment by gossebouma** [comment]
> I could imagine a version of UD where all of these are xcomp, but that is not the case in almost any UD treebank I looked at in any language:

In Dutch, comparable cases (beschouwen als X, consider as, beschrijven als X, portray as) are analysed as xcomp(beschouwen,X) and mark(X,als) ('als', 'as', is labeled with POS SCONJ), e.g. 
http://universal.grew.fr/?custom=63e4af4353b35

So in that sense, Dutch follows the proposal of Nathan.

**Comment by nschneid** [comment]
(Just heard a direct object vs. predicative complement joke (in the TV show *Benson*): "[walks into a cluttered kitchen] I don't believe it. [He] left me a big mess!" "Well, it's only fair. That's how he found you!")

**Comment by LarsAhrenberg** [comment]
Nice one! But what is the proper way to handle puns like that in UD, given that we don't have disjunctive deprels: choose one and ignore the other, or enter the sentence twice with different analyses?

**Comment by nschneid** [comment]
Guidelines clarified: https://github.com/UniversalDependencies/docs/compare/2a3ac4d374932411df18d9e3554475147ce0cccb...pages-source

**Comment by amir-zeldes** [comment]
Noted, but I don't see who will actually implement this in the corpora, given that most of them do not annotate prepositions that can be interpreted like "as" with xcomp, and that it's not trivial to disambiguate... see my small sample [above](https://github.com/UniversalDependencies/UD_English-EWT/issues/387#issuecomment-1422987399)
